boards/imix/src/components/fxos8700.rs

```diff
@@ -25,9 +25,11 @@
 use capsules::fxos8700cq;
 use capsules::ninedof::NineDof;
 use capsules::virtual_i2c::{I2CDevice, MuxI2C};
-use hil;
 use kernel::capabilities;
 use kernel::component::Component;
+use kernel::create_capability;
+use kernel::hil;
+use kernel::static_init;
 use kernel::Grant;
 
```

Comment:
These imports could be a one line

Reply:
Haha you know I'm not going to make that change 😄  If rustfmt decided on it then great, but otherwise I think the one per line is much easier to read.